wkcuber/api/View.py

```diff
@@ -99,6 +123,20 @@ def open(self):
             self._is_opened = True
         return self
 
+    def _check_chunk_size(self, chunk_size):
+        file_dim = (self.header.file_len * self.header.block_len,) * 3
+
+        assert chunk_size is not None
+
+        if 0 in chunk_size:
+            raise AssertionError(
+                f"The passed parameter 'chunk_size' {chunk_size} contains at least one 0. This is not allowed."
+            )
+        if (np.array(chunk_size) % file_dim).any():
+            raise AssertionError(
+                f"The passed parameter 'chunk_size' {chunk_size} must be a multiple of the file size {file_dim}"
```

Comment:
Does that mean when using files with cube length 1024, only multiples of 1024 are valid chunk sizes? Given the structure of WKW, I would rather enforce the chunk size to be a power of 2 and ≥ 32.

Reply:
I changed the behavior of this method as you suggested.